mapper.go

```diff
@@ -143,11 +151,15 @@ func QueryTable(db *sqlx.DB, ts TableSchema) ([]map[string]interface{}, error) {
 			default:
 				log.Println("unhandled type")
 			}
+
+			// check to see if there is a priarmy on this key
 			if ts.ColumnNames()[i] == ts.PrimaryKey {
 				// If this column is the PK; assign it again to a special key in our map
 				m["__pk"] = fmt.Sprintf("%v", m[ts.ColumnNames()[i]])
 			}
 
+			//m = setPrimaryKey(m, ts.PrimaryKey)
```

Comment:
yeah i need to add that function just mapping out in my head how we should set Primary key. That will remove the check we do on line 156.

Basically check if a column is the Primary key, if not then generate the Guid as you mentioned in some other thing.